src/Components/App/App.js

```diff
@@ -217,296 +214,36 @@ class App extends Component {
             }}
           />
 
-          <Route
-            exact
-            path='/:category/:type'
-            render={({ match }) => {
-              if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'blush'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='blush'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'eyeshadow'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='eyeshadow'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'foundation'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='foundation'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'mascara'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='mascara'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'bronzer'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='bronzer'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'eyeliner'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='eyeliner'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'lipstick'
-              ) {
-                return (
-                  <Items
-                    data={this.state.vegan}
-                    type='lipstick'
-                    category='vegan'
-                  />
-                );
-              } else if (
-                match.params.category === 'vegan' &&
-                match.params.type === 'lip_liner'
-              ) {
+          <Route 
```

Comment:
I'm grateful that you've taken the time to refactor this. Though, I'm 100% confused about why the first time we tried to implement it, it didn't work.